analysis/import/SCMLogParser/src/main/kotlin/de/maibornwolff/codecharta/importer/scmlogparser/input/VersionControlledFile.kt

```diff
@@ -48,7 +46,7 @@ class VersionControlledFile internal constructor(
         val type = modification.type
         when (type) {
             Modification.Type.DELETE -> markedDeleted = true
```

Comment:
This is actually not required anymore. No code is going to access `markedDeleted`.